### ISSUE TITLE
Videos are saved as root

### ISSUE THREAD

**Issue by JohannesHome** [bahmutov/demo-docker-cypress-included#2]
Doing `./cy-run.sh` creates `./cypress/videos/spec.js.mp4` as root. 

Not sure if this is a bug, or if this is intentional?

**Comment by bahmutov** [comment]
Yes, after installing Chrome browser, the current user in the container is `root`. But can I ask how you are running the container (and on what operating system)?

For my Mac, despite what https://medium.com/redbubble/running-a-docker-container-as-a-non-root-user-7d2e00f8ee15 says, when I run this image, I get the files correctly attributed to me

```
$ docker run -it -v $PWD:/e2e -w /e2e cypress/included:3.4.0
...
cypress/videos are created

$ ls -l cypress/videos
total 32
-rw-r--r--  1 gleb  staff  15498 Jul 18 13:02 spec.js.mp4

$ ls -l cypress/videos/spec.js.mp4 
-rw-r--r--  1 gleb  staff  15498 Jul 18 13:02 cypress/videos/spec.js.mp4
```

so the folder and the video are owned by me: `gleb:staff`

**Comment by bahmutov** [comment]
closing as hopefully resolved
